metadata_service/config.py

```diff
@@ -39,6 +39,21 @@ class Config:
     # The relationalAttribute name of Atlas Entity that identifies the database entity.
     ATLAS_DB_ATTRIBUTE = 'db'
 
+    # Configurable dictionary to influence format of column statistics displayed in UI
+    # The key is name of statistic and a value is a dictionary with optional keys:
+    #   new_name - how to rename statistic (if absent proxy will default to old name)
+    #   format - how to format numerical statistics (if absent, proxy will default to {})
+    #   drop - should given statistic not be displayed in UI (if absent, proxy will keep it)
+    ATLAS_STATISTICS_FORMAT_SPEC = {
+        'standard deviation': dict(format='{:,.2f}'),
+        'mean': dict(format='{:,.2f}'),
+        'max': dict(new_name='maximum', format='{:,.2f}'),
```

Comment:
Can we please not make this specific to Atlas? so that other proxies can also use this ?
`STATISTICS_FORMATTER` or `STATISTICS_FORMAT_SPEC` ? and then add this in the configurations doc ?

Reply:
done